### PR TITLE
Fix ConnectionLost -> Connect for Windows

### PR DESCRIPTION
**Pull request by AskBojesen**
Reproduce the error:
1. Connect
2. ConnectLost
3. Connect

Thereafter ConnectionChanged event was fired twice subsequently

@janusw : Yet another small PR for the Windows stability

### CODE REVIEW

File: Source/Plugin.BLE/Windows/Adapter.cs [hunk 1]
```diff
@@ -144,11 +144,8 @@ private void Device_ConnectionStatusChanged(BluetoothLEDevice nativeDevice, obje
                 }
 
                 // fire the correct event (DeviceDisconnected or DeviceConnectionLost)
-                HandleDisconnectedDevice(isNormalDisconnect, disconnectedDevice);
-                if (isNormalDisconnect)
-                {
-                    nativeDevice.ConnectionStatusChanged -= Device_ConnectionStatusChanged;
-                }
+                HandleDisconnectedDevice(isNormalDisconnect, disconnectedDevice);
```
Comment (janusw) [inline Source/Plugin.BLE/Windows/Adapter.cs:147]: There is no real change in line 147. Apparently you're adding unnecessary whitespace at the end.

**Comment by AskBojesen** [comment]
I ended up with a quite big change in the adapter.cs , but I am really confident this now solves my issues since I tested a lot in the two cases:

1. Connect -> Disconnect ->Connect -> Disconnect ->Connect -> Disconnect 

2. Connect -> LostConnection ->Connect -> LostConnection -> Connect -> LostConnection

**Comment by AskBojesen** [comment]
@janusw: Can you look at this PR?

**Review by janusw — APPROVED** [review]
I'm not able to judge if this does the right thing in every circumstance, but I don't see anything obviously wrong with your code either.
So if you tested it thoroughly, the PR is ok with me.
